lib/report-builder/static.ts

```diff
@@ -129,6 +129,24 @@ export class StaticReportBuilder {
         const isPreviouslySkippedTest = isImgSkipped && getTestFromDb<DbTestResult>(this._dbClient, formattedResult);
 
         if (!ignoredStatuses.includes(testResultWithImagePaths.status) && !isPreviouslySkippedTest) {
+            if (this._reporterConfig.badgeFormatter && typeof this._reporterConfig.badgeFormatter === 'function') {
+                if (testResultWithImagePaths.attachments) {
```

Comment:
These 2 nested "if"s can be rewrited to:
```js
if (typeof this._reporterConfig.badgeFormatter === 'function' && testResultWithImagePaths.attachments) {
...
}
```

Reply:
Fixed